doc/rtd/reference/datasources/vmware.rst

```diff
@@ -399,28 +399,28 @@ this datasource using the GuestInfo keys transport:
 
       govc vm.power -off "${VM}"
 
-#. Export the environment variables that contain the ``cloud-init`` metadata
-   and user data:
+#. Export the environment variables that contain the ``cloud-init`` meta-data
+   and user-data:
 
    .. code-block:: shell
 
-      export METADATA=$(gzip -c9 <metadata.yaml | { base64 -w0 2>/dev/null || base64; }) \
+      export METADATA=$(gzip -c9 <meta-data.yaml | { base64 -w0 2>/dev/null || base64; }) \
            USERDATA=$(gzip -c9 <userdata.yaml | { base64 -w0 2>/dev/null || base64; })
 
-#. Assign the metadata and user data to the VM:
+#. Assign the meta-data and user-data to the VM:
 
    .. code-block:: shell
 
        govc vm.change -vm "${VM}" \
-       -e guestinfo.metadata="${METADATA}" \
-       -e guestinfo.metadata.encoding="gzip+base64" \
+       -e guestinfo.meta-data="${METADATA}" \
+       -e guestinfo.meta-data.encoding="gzip+base64" \
        -e guestinfo.userdata="${USERDATA}" \
        -e guestinfo.userdata.encoding="gzip+base64"
```

Comment:
should these be updated as well?

Reply:
I think that this would require changes to an external project, so not today.